test/integration/features/F001_Valid_Change_Events.feature

```diff
@@ -55,3 +55,41 @@ Feature: F001. Ensure valid change events are converted and sent to DOS
     When the Changed Event is sent for processing with "valid" api key
     Then the Changed Request is accepted by Dos
     And the Dentist changes with service type id is captured by Dos
+
+@complete @splunk
```

Comment:
Can each of these tests be tagged with `@cloudwatch_queries` so it gets run on the development pipeline